pandas/tests/extension/test_arrow.py

```diff
@@ -1266,7 +1214,12 @@ def test_invalid_other_comp(self, data, comparison_op):
 
 def test_arrowdtype_construct_from_string_type_with_unsupported_parameters():
     with pytest.raises(NotImplementedError, match="Passing pyarrow type"):
-        ArrowDtype.construct_from_string("timestamp[s, tz=UTC][pyarrow]")
+        ArrowDtype.construct_from_string("not_a_real_dype[s, tz=UTC][pyarrow]")
+
+    # but as of GH#50689, timestamptz is supported
```

Comment:
In a follow up could you make this it's own test? (Nit: doesn't really fit the name of the test"

Reply:
fair enough, will do